packages/block-library/src/navigation/index.php

```diff
@@ -611,6 +582,11 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 
 	// Load inner blocks from the navigation post.
 	if ( array_key_exists( 'ref', $attributes ) ) {
+		if ( in_array( $attributes['ref'], $seen_ref, true ) ) {
+			return '';
+		}
+		$seen_ref[] = $attributes['ref'];
```

Comment:
This could be smarter and keep track of the parent rather than a flat array so you could have two identical sibling navigation blocks, but Navigation blocks probably shouldn't be nested inside other navigation blocks in the first place, so it's probably fine this way.

Reply:
This ended up not being fine. https://github.com/WordPress/gutenberg/issues/47085